README.md

```diff
@@ -75,6 +75,10 @@ If you are interested in contributing to the new AWS SDK for Rust, please take a
 
 This alpha SDK currently does not provide support for every AWS service. You can see all the services currently supported on [AWS_SERVICES_SUPPORTED](AWS_SERVICES_SUPPORTED.md)
 
+## Supported Rust Versions
+
+The SDK currently requires a minimum of Rust 1.51.0, and is not guaranteed to build on compiler versions earlier than that. While we are still in alpha, we will be keeping the minimum compiler version two releases behind the latest stable release (so if the latest stable is 1.53, we will be on 1.51), but we are not making any guarantees around this at present. Increases in minimum required Rust version will be called out in the Release Notes for new releases of the SDK.
```

Comment:
it's actually 1.52.0 because of a usage of some doc lints. We can roll it back if we want

Reply:
I'd include the string `MSRV` somewhere, that's what I'd search for if I was trying to figure this out